src/Storefront/Resources/app/storefront/src/plugin/forms/form-cms-handler.plugin.js

```diff
@@ -76,6 +76,9 @@ export default class FormCmsHandler extends Plugin {
     }
 
     _submitForm() {
+        if (this.formSubmittedByCaptcha) {
+            return;
+        }
         this.$emitter.publish('beforeSubmit');
```

Comment:
What about other extensions that might listen on this event being submitted? Is it still submitted correctly when `this.formSubmittedByCaptcha` is "truethy"?

I just want to be extra careful that we don't introduce any last minute extension breaks with the final release 😅

Reply:
Only the login subscribes to this event, and this works well